zap/src/output/luau/server.rs

```diff
@@ -70,6 +70,42 @@ impl<'a> ServerOutput<'a> {
 		}
 	}
 
+	fn push_event_loop(&mut self) {
+		self.push_line("");
+
+		if self.config.manual_event_loop {
+			let send_events = self.config.casing.with("SendEvents", "sendEvents", "send_events");
+
+			self.push_line(&format!("local function {send_events}()"));
+		} else {
+			self.push_line("RunService.Heartbeat:Connect(function()");
+		}
+
+		self.push_line(
+			r#"	for player, outgoing in player_map do
+		if outgoing.used > 0 then
+			local buff = buffer.create(outgoing.used)
+			buffer.copy(buff, 0, outgoing.buff, 0, outgoing.used)
+
+			reliable:FireClient(player, buff, outgoing.inst)
+
+			outgoing.buff = buffer.create(64)
+			outgoing.used = 0
+			outgoing.size = 64
+			table.clear(outgoing.inst)
+		end
+	end"#,
```

Comment:
Same comments as for the client here - split this into `self.push_line`, `self.indent` etc